src/SourceBuild/content/build.sh

```diff
@@ -39,6 +39,7 @@ usage()
   echo "  --excludeCIBinarylog            Don't output binary log (short: -nobl)"
   echo "  --prepareMachine                Prepare machine for CI run, clean up processes after build"
   echo "  --use-mono-runtime              Output uses the mono runtime"
+  echo "  --dev                           Use -dev or -ci versioning instead of Microsoft official build versions. Ignored when --source-only is specified."
```

Comment:
Now that we have multiple non source-only settings, thoughts on moving these to a separate section much like we have the source-only settings?